src/s2japp/Application.cc

```diff
@@ -166,11 +168,13 @@ Application::ExitCode Application::handleRun() {
 }
 
 void Application::initializeLogger() {
-    if (settings_.loggerPath == "-")
+    if (settings_.loggerPath == "-") {
         logger_ = std::make_shared<s2j::logger::FDLogger>(2 /* stderr */);
-    else if (!settings_.loggerPath.empty())
+    }
+    else if (!settings_.loggerPath.empty()) {
```

Comment:
nie w tej samej linii co klamra?

Reply:
Pewnie do poprawy w .clang-format (cena za automatyczne formatowanie...)

Reply:
A czekaj, mówisz o:
```
}
else if (...) {
```
vs
```
} else if (...) {
```
?

Reply:
tak

Reply:
No, to ogólnie kwestia stylu jaki chcemy przyjąć, ja wolę to pierwsze (i tak też pisałem clang-format). Jak chcemy, możemy zmienić, odpowiada opcja `BraceWrapping`, a dokładniej pewnie `AfterControlStatement` wewnątrz niej (https://clang.llvm.org/docs/ClangFormatStyleOptions.html). Jak chcemy możemy zmienić (ale to trochę niezwiązane z tym co jest teraz ;p)

Reply:
Ok, to proponuję zmienić potem, jeśli nie masz nic przeciwko. `AfterControlStatement: false` jest IMO bardziej symetryczne, i widać że ten `else` jest kontynuacją poptrzedniego ifa, a nie niezależną konstrukcją.